src/decision/src/LidarDecision.cpp

```diff
@@ -31,6 +32,143 @@ void LidarDecision::scanCallBack(const sensor_msgs::LaserScan::ConstPtr& raw_sca
     // Deal with new messages here
```

Comment:
This is the callback function that is called every time you receive a new lidar scan, but there's nothing here? How are you getting the lidar scan and triggering the rest of your decision process? (You probably want to call `manage_twist` from here)